src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsArticles.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box } from "@artsy/palette"
+import { ArticleItem } from "Apps/Artist/Routes/Articles/ArtistArticle"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Col, Row } from "Styleguide/Elements/Grid"
+
+import { ArtworkDetailsArticles_artwork } from "__generated__/ArtworkDetailsArticles_artwork.graphql"
+
+export interface ArtworkDetailsArticlesProps {
+  artwork: ArtworkDetailsArticles_artwork
+}
+
+export class ArtworkDetailsArticles extends React.Component<
```

Comment:
Same here -- if not using `React.Class` aspects would be better if it was a `React.SFC`